### DISCUSSION TITLE
Custom Gtag source

### DISCUSSION THREAD

**Discussion by StijnCoolen** [johannschopplich/nuxt-gtag#15] [Q&A] [answered]
Is there any way to add/overwrite the src of the script?

**Comment by johannschopplich** [comment]
Nope. What's your use-case here? Isn't Google Analytics always hosted from the same URL?

**Reply by StijnCoolen** [reply]
Might be very specific, but you are able to host your own script. I was able to use `source` in @gtm-support/vue-gtm

**Reply by johannschopplich** [reply, marked as answer]
Why would you want to host the official `gtag.js` manually? One would have to download it and host it along a project. This doesn't make sense in my opinion. Any updates to the official `gtag.js` would be missed.
If you have any use-cases, please share them. I can't find one on my own. 😊
